### ISSUE TITLE
Resource Report - 1.4.10 Reflow - Content must be viewed by scrolling in two dimensions at 400%

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9761]
As a user reliant on the highest zoom setting to perceive content, all text and components should be visible and accessible at 400% zoom and should reflow into a single column. No content should be obscured by other content or cut off by the edge of the page. Users should not be forced to scroll in two dimensions to view content. 

**Linked work item:**
#9663 

**Expected Result:**
Content should reflow into a single column at 400%, with all content on the page visible and accessible at this zoom level. User should be able to scroll in a single dimension to view the entire page.

**Actual Result:**
Content must be viewed by scrolling in two dimensions at 400%, and still some content is cut off the right-hand edge of the page (Report Date, 'Hide Null Values' slider and Additional Information).

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/6166ca63-d231-4303-b9fe-18e1ad72205e)

**Comment by johnatawnclementawn** [comment]
Will still need to be addressed after #9741

**Comment by johnatawnclementawn** [comment]
@imogenroberts please pull latest and RETEST

**Comment by imogenroberts** [comment]
RETEST: The user can zoom all the way up to 400% without having to scroll in two dimension to see all of the content; all content reflows into a single column.

**Evidence:**
![Uploading Resource Report - 1.4.10 Fixed.gif…]()
